gui/cult-lists.lua

```diff
@@ -0,0 +1,1104 @@
+﻿-- cult-lists 1.1
+--
+-- Prints all deities and religions in fort and lists all their worshippers by name.
+-- Inspired by Fortunawhisk's Monotheism and Robob27's Adjust-Piety
+-- Thanks to myk002, Rumrusher, ab9rf and coskerstrike for tips with LUA and DFhack API.
+-- Special thanks to myk002 for the crash course on overlay widgets
+--
+------------------------------------------------------------------------------------------
+-- Noticed that this was an open fork, but hadn't been touched in months, so decided to have a shot at getting it finished
+-- I am not super familiar with lua or C/C++ but will see how thinbgs go.
+-- Doesn't seem like there are many problems with the script itself just needed to be formatted
+-----------------------------------------------------------------------------------------
+
+-- NOTES:
+
+-- things that would greatly simplify this script's horrible spaghetti coding:
+    -- Wrapping label widget taking tokens so I can properly colour words in sentences
+    -- Deity's entity links vector leading to their associated civs (is empty) so i don't depend on so many iterations
+    -- force-opening DF screens, such as unit screen, on unit selection so i don't have to open unit screen separately (doesn't help the code tho)
+
+--TO-DO
+    -- zoom on unit -- can't be done yet
+    -- never touch LUA script again
+
+------------------------------------------------------------------------------------------
+
+-- error handling
+
+if not dfhack.isMapLoaded() then
+    qerror('This script requires fort mode to be loaded') --(qerror might be typo, and should probably be error)
```

Comment:
Not a typo. https://docs.dfhack.org/en/stable/docs/dev/Lua%20API.html#exception-handling